plugins/grpc-plugin/src/main.rs

```diff
@@ -53,17 +54,7 @@ async fn main() -> Result<()> {
         None => return Ok(()),
     };
 
-    let bind_port = match plugin.option(&OPTION_GRPC_PORT).unwrap() {
-        Some(port) => port,
-        None => {
-            log::info!("'grpc-port' options i not configured. exiting.");
-            plugin
-                .disable("Missing 'grpc-port' option")
-                .await?;
-            return Ok(())
-        }
-    };
-
+    let bind_port : i64 = plugin.option(&OPTION_GRPC_PORT).unwrap();
```

Comment:
I'd still want to be able to disable the plugin when starting the node. How about not starting when the port number is 0 or negative?

Reply:
Better a flag instead

Reply:
there's already a `disable-plugin` flag.